src/applications/move_reminder.c

```diff
@@ -0,0 +1,1708 @@
+    return MOVE_REMINDER_STATE_PRINT_MESSAGE_BOX_TEXT;
+}
+
+static int MoveReminder_ForgetThisMove_No(MoveReminderController *controller)
+{
+    controller->nextState = MOVE_REMINDER_STATE_FADE_TO_SUMMARY_SCREEN;
+    return MOVE_REMINDER_STATE_INIT;
+}
+
+static void MoveReminder_InitSpriteSystem(MoveReminderController *controller)
+{
+    SpriteResourceCapacities capacities = {
+        .asStruct = {
+            .charCapacity = 11,
+            .plttCapacity = 2,
+            .cellCapacity = 4,
+            .animCapacity = 4,
+            .mcellCapacity = 0,
+            .manimCapacity = 0,
+        },
+    };
+
+    VramTransfer_New(64, HEAP_ID_MOVE_REMINDER);
+
+    controller->spriteSystem = SpriteSystem_Alloc(HEAP_ID_MOVE_REMINDER);
+    controller->spriteManager = SpriteManager_New(controller->spriteSystem);
+
+    RenderOamTemplate renderOamTemplate = {
+        .mainOamStart = 0,
+        .mainOamCount = 128,
+        .mainAffineOamStart = 0,
+        .mainAffineOamCount = 32,
+        .subOamStart = 0,
+        .subOamCount = 128,
+        .subAffineOamStart = 0,
+        .subAffineOamCount = 32,
+    };
+
+    CharTransferTemplateWithModes charTransferTemplate = {
+        .maxTasks = 13,
+        .sizeMain = 1024 * 128,
+        .sizeSub = 1024 * 16,
+        .modeMain = GX_OBJVRAMMODE_CHAR_1D_32K,
+        .modeSub = GX_OBJVRAMMODE_CHAR_1D_32K,
+    };
+
+    SpriteSystem_Init(controller->spriteSystem, &renderOamTemplate, &charTransferTemplate, 32);
+    SpriteSystem_InitSprites(controller->spriteSystem, controller->spriteManager, NELEMS(controller->managedSprites));
+    SpriteSystem_InitManagerWithCapacities(controller->spriteSystem, controller->spriteManager, &capacities);
+    GXLayers_EngineAToggleLayers(GX_PLANEMASK_OBJ, TRUE);
+}
+
+static void MoveReminder_FreeSpriteSystem(MoveReminderController *controller)
+{
+    for (u32 i = 0; i < NELEMS(controller->managedSprites); i++) {
+        Sprite_DeleteAndFreeResources(controller->managedSprites[i]);
+    }
+
+    SpriteSystem_FreeResourcesAndManager(controller->spriteSystem, controller->spriteManager);
+    SpriteSystem_Free(controller->spriteSystem);
+}
+
+static void MoveReminder_UpdateSpritesAnim(MoveReminderController *controller)
+{
+    for (u16 i = 0; i < NELEMS(controller->managedSprites); i++) {
+        Sprite_UpdateAnim(controller->managedSprites[i]->sprite, FX32_ONE);
+    }
+}
+
+static void MoveReminder_LoadSpriteResources(MoveReminderController *controller, NARC *narc)
+{
+    SpriteSystem_LoadCharResObjFromOpenNarc(controller->spriteSystem, controller->spriteManager, narc, 5, FALSE, NNS_G2D_VRAM_TYPE_2DMAIN, 47272);
+    SpriteSystem_LoadCharResObjFromOpenNarc(controller->spriteSystem, controller->spriteManager, narc, 8, FALSE, NNS_G2D_VRAM_TYPE_2DMAIN, 47273);
+    SpriteSystem_LoadCharResObjFromOpenNarc(controller->spriteSystem, controller->spriteManager, narc, 2, FALSE, NNS_G2D_VRAM_TYPE_2DMAIN, 47274);
```

Comment:
Polish: if this only expects a single narc replace values with narc member indexes. This applies to the calls lower in this function as well